soh/soh/Enhancements/timesaver_hook_handlers.cpp

```diff
@@ -279,7 +279,7 @@ void TimeSaverOnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, va_li
                         *should = false;
                         RateLimitedSuccessChime();
                         break;
-                    }
+                    }//is there a reason these aren't combined?
```

Comment:
What does this comment mean exactly? Arent these cases separate?

Reply:
Unless Pepper0ni was talking about the next 4, which are handled exactly the same?

Reply:
I was. The question was why not stack the 4 up like the lower bunch. I didn't want to just do it in case there was some edge case or later development planned.

Reply:
Just my two cents, but I would prefer if we didn't commit questions like this into the source. Some of these comments have already been merged and now live in the code without a way to find them unless someone stumbles upon them.

Either make a formal TODO, or propose the question as a comment within the github pull-request for better discussions.

To answer the question, I don't see a reason why they couldn't be combined, so long as they aren't combined with the ones below that also play a chime sound.

Reply:
The chime sound is actually not doing anything at all right now, because I couldn't find a good way to have it play that was consistent with vanilla. So all of these could be combined, unless we figure the chime out

Reply:
Combine them and remove the comment, and I think we can merge it.